README.md

```diff
@@ -1,11 +1,11 @@
-[![Build Status](https://travis-ci.org/Microsoft/TypeScript.svg?branch=master)](https://travis-ci.org/Microsoft/TypeScript)
+[![Build Status](https://img.shields.io/travis/Microsoft/TypeScript.svg?logo=travis)](https://travis-ci.org/Microsoft/TypeScript)

```

Comment:
```suggestion
[![Build Status](https://travis-ci.org/Microsoft/TypeScript.svg?branch=master)](https://travis-ci.org/Microsoft/TypeScript)
```

Reply:
Turns out that **Shields.io** now supports Travis branch builds too:

```suggestion
[![Build Status](https://img.shields.io/travis/Microsoft/TypeScript/master.svg?logo=travis)](https://travis-ci.org/Microsoft/TypeScript)
```

Reply:
Why `logoColor=white`?

Reply:
It looks better than without it: [<img alt="Build Status" src="https://img.shields.io/travis/Microsoft/TypeScript/master.svg?logo=travis-ci" align="top"/>](https://travis-ci.org/Microsoft/TypeScript)

Reply:
Huh, why does it turn blue when you pass it the master branch? Without it the default is

[![Build Status](https://img.shields.io/travis/Microsoft/TypeScript.svg?logo=travis)](https://travis-ci.org/Microsoft/TypeScript)
 

Reply:
Because I used [the **simple-icons** `travis-ci`](https://simpleicons.org/?q=travis%20ci) logo, instead of the **shields.io** `travis` logo.

Reply:
Oh, it's a difference of `travis` vs `travis-ci`. I wasn't aware shields added their own logos since it also uses simpleicons. In that case, `travis` should have probably been kept anyways since it aligns with the style of the npm logo - which shields has no way of setting to the simpleicons version since they're both named the same.